cypress/integration/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -250,6 +250,26 @@ function testWorkflow(registry: string, repo: string, name: string) {
       cy.contains('button', 'Restub').click();
       cy.contains('button', 'Publish').should('be.disabled');
     });
+    it('hide and un-hide a version', () => {
+      goToTab('Versions');
+      // hide
+      cy.get('[data-cy=refreshButton]').click();
+      cy.get('[data-cy=versionRow]').last().contains('button', 'Actions').should('be.visible').click();
+      cy.contains('button', 'Edit').click();
+      cy.contains('div', 'Hidden:').within(() => {
+        cy.get('[name=checkbox]').click();
+      });
+      cy.contains('button', 'Save Changes').click();
+      cy.get('[data-cy=hidden]').should('have.length', 1);
+      // un-hide
+      cy.get('[data-cy=versionRow]').last().contains('button', 'Actions').should('be.visible').click();
+      cy.contains('button', 'Edit').click();
+      cy.contains('div', 'Hidden:').within(() => {
+        cy.get('[name=checkbox]').click();
+      });
+      cy.contains('button', 'Save Changes').click();
+      cy.get('[data-cy=hidden]').should('not.exist');
```

Comment:
The `data-cy` tag in the hidden box in the workflow options is `hidden`, whereas in the tool options it is `hiddenCheck`. I'm thinking if these were the same it would make it easier to write a function to do the test (and make sense?).

Reply:
The box I am referring to when I say "hidden box" is the one highlighted in this image,
![Screenshot from 2022-08-19 17-22-52](https://user-images.githubusercontent.com/85577755/185709309-bc9d6c1f-f90a-479a-9486-9e506eddcb7e.png)